solver/llbsolver/proc/index.go

```diff
@@ -0,0 +1,71 @@
+package proc
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/containerd/containerd/platforms"
+	"github.com/moby/buildkit/exporter/containerimage/exptypes"
+	"github.com/moby/buildkit/frontend"
+	"github.com/moby/buildkit/solver"
+	"github.com/moby/buildkit/solver/llbsolver"
+	ocispecs "github.com/opencontainers/image-spec/specs-go/v1"
+)
+
+func ForceIndexProcessor(ctx context.Context, result *frontend.Result, s *llbsolver.Solver, j *solver.Job) (*frontend.Result, error) {
```

Comment:
Add comment to this func. I didn't initially understand what it did based on name.